ci/trim_pipeline.py

```diff
@@ -7,7 +7,7 @@
 # the Business Source License, use of this software will be governed
```

Comment:
The `ci` directory doesn't presently have shared scripts at the top level and I'd like to keep it that way! Could you move to `materialize.ci_util.trim_pipeline`? You'll need to make the following changes:

  * `misc/python/materialize/ci_util.py` -> `misc/python/materialize/ci_util/__init__.py`
  * `ci/nightly/trim_pipeline.py` -> `misc/python/materialize/ci_util/trim_pipeline.py`